src/yorkie.ts

```diff
@@ -45,14 +45,6 @@ export {
   Document,
   ChangeInfo,
 } from '@yorkie-js-sdk/src/document/document';
-export {
-  Observer,
-  Observable,
-  NextFn,
-  ErrorFn,
-  CompleteFn,
-  Unsubscribe,
-} from '@yorkie-js-sdk/src/util/observable';
```

Comment:
I removed this line because I think we don't need to expose these functions(which have @internal tag) to SDK user and in the docs.

Reply:
Since the internal tag has been removed, it seems fine to export as usual. In CodePair, we're using [Unsubscribe](https://github.com/yorkie-team/codepair/blob/9a94d3b78511adadc168689821176716175fb8d1/src/components/Editor/mime/application/whiteboard/hooks/useMultiPlayerState.ts#L21). How about considering removing it after confirming if it's unnecessary?

Reply:
Aha sure! I will remove it in the separate PR!